Automattic/wp-calypso
# Stats: Adding an icon to indicate titles with links

#### Proposed Changes

* adding a chevron icon with hover transformation to indicate titles that are also links (applicable to a highlights component and modules with horizontal bars)

#### Testing Instructions



* open `Stats` page and verify that components with horizontal bars on the `Traffic` page have chevron icons and are links redirecting to the details page
* navigate to `Insights` page and verify that `2022 in review` highlight component also has a chevron and link to more details

Before:
![Screenshot 2022-12-08 at 10 25 15 AM](https://user-images.githubusercontent.com/112354940/206299800-3da58f3a-a6c2-4864-a689-8fc1b33c88f6.png)

![Screenshot 2022-12-08 at 10 25 37 AM](https://user-images.githubusercontent.com/112354940/206299831-8cb19306-f5c3-4101-916b-bc8b5c80a0ec.png)

After:
![Screenshot 2022-12-08 at 10 24 35 AM](https://user-images.githubusercontent.com/112354940/206299888-d2bdc547-3df7-46c0-bdb8-9b870e178ba8.png)

![Screenshot 2022-12-08 at 10 25 52 AM](https://user-images.githubusercontent.com/112354940/206299932-b67cbd89-53bc-431f-8e79-6e4acaf3f252.png)


#### Pre-merge Checklist





Related to #70849
